Warn or allow the user to use different computers for the electronic calculation and DDEC

Using different computers for the `Cp2kDdecWorkChain` (even if it was actually the same machine with different constrained nodes, e.g., the fidis and gacrux EPFL cluster) resulted in the following error:
```
10/13/2019 12:05:18 AM <13001> aiida.orm.nodes.process.calculation.calcjob.CalcJobNode: [ERROR] iteration 1 of do_upload excepted, retrying after 20 seconds
Traceback (most recent call last):
  File "/home/daniele/aiida1/aiida_core/aiida/engine/utils.py", line 176, in exponential_backoff_retry
    result = yield coro()
  File "/home/daniele/anaconda3/envs/aiida1/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run
    value = future.result()
  File "/home/daniele/anaconda3/envs/aiida1/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result
    raise_exc_info(self._exc_info)
  File "<string>", line 4, in raise_exc_info
  File "/home/daniele/anaconda3/envs/aiida1/lib/python3.6/site-packages/tornado/gen.py", line 1069, in run
    yielded = self.gen.send(value)
  File "/home/daniele/aiida1/aiida_core/aiida/engine/processes/calcjobs/tasks.py", line 76, in do_upload
    raise Return(execmanager.upload_calculation(node, transport, calc_info, script_filename))
  File "/home/daniele/aiida1/aiida_core/aiida/engine/daemon/execmanager.py", line 223, in upload_calculation
    'calculation {}'.format(node.pk))
OSError: It is not possible to create a symlink between two different machines for calculation 35887
10/13/2019 12:05:45 AM <13001> aiida.execmanager: [WARNING] tried to create path /scratch/ongari/aiida_run/5d/4a/6c1c-0bf5-4c5a-98e4-0ffc24131123 but it already exists, moving the entire folder to /scratch/ongari/aiida_run/lost+found/5d4a6c1c-0bf5-4c5a-98e4-0ffc24131123
10/13/2019 12:05:46 AM <13001> aiida.engine.transports: [ERROR] Exception whilst using transport:
Traceback (most recent call last):
  File "/home/daniele/aiida1/aiida_core/aiida/engine/transports.py", line 106, in request_transport
    yield transport_request.future
  File "/home/daniele/aiida1/aiida_core/aiida/engine/processes/calcjobs/tasks.py", line 76, in do_upload
    raise Return(execmanager.upload_calculation(node, transport, calc_info, script_filename))
  File "/home/daniele/aiida1/aiida_core/aiida/engine/daemon/execmanager.py", line 223, in upload_calculation
    'calculation {}'.format(node.pk))
OSError: It is not possible to create a symlink between two different machines for calculation 35887
```

This comes from the fact that aiida can not copy the symlink of ELECTRON_DENSITY.cube generated by CP2K in the DDEC calculation folder. 
The error message is very cryptic an one can find it only in the daemon log. 
Let's consider about:

1) warning the user that he can not use the `Cp2kDdecWorkChain` with codes from two different computers (even if they are the same machine in this case!)
or
2) allowing the transfer of cube files from one machine to the other. This should use the local computer as a bridge and can be problematic considering that cube files are typically 300MB-1GB files. 

Suggestions?

--- a/aiida_ddec/calculations/__init__.py
+++ b/aiida_ddec/calculations/__init__.py
@@ -175,8 +175,16 @@ class DdecCalculation(CalcJob):
                 charge_density_folder.get_remote_path(),
                 'aiida-ELECTRON_DENSITY-1_0.cube',
             )
-            symlink = (comp_uuid, remote_path, 'valence_density.cube')
-            calcinfo.remote_symlink_list.append(symlink)
+            copy_info = (comp_uuid, remote_path, 'valence_density.cube')
+            if self.inputs.code.computer.uuid == comp_uuid:  # if running on the same computer - make a symlink
+                calcinfo.remote_symlink_list.append(copy_info)
+            else:  # if not - copy the folder
+                self.report(
+                    f"Warning: Transferring cube file {charge_density_folder.get_remote_path()} from " +
+                    f"computer {charge_density_folder.computer.label} to computer {self.inputs.code.computer.label}. " +
+                    "This may put strain on your network."
+                )
+                calcinfo.remote_copy_list.append(copy_info)
 
         codeinfo = CodeInfo()
         codeinfo.cmdline_params = []

